templates/signup.html

```diff
@@ -1,11 +1,46 @@
 {% extends "auth.html" %}
 
+{% if request.args.get('teacher') %}
+{% set is_teacher = request.args.get('teacher') == 'true' %}
+{% else %}
+{% set is_teacher = None %}
+{% endif %}
+
 {% block main %}
 <div class="w-full mx-auto flex flex-col items-center gap-4">
     <div class="bg-white shadow-md rounded-lg rounded py-4 px-0 md:px-4 lg:px-8">
+        {% if is_teacher == None %}
```

Comment:
Why don't we simplify this with `{% if not request.args.get('teacher') %}`? This would simplify our code, with the following flow in mind "if there is no url parameter with the name 'teacher' show the selector, otherwise show the signup form".

Reply:
I do agree that it's a little more descriptive, but I did not want to include the "complexity" of how this is determined in the template

Reply:
yeah this is a bit of a trade off between readability and maintainability but I agree with @TiBiBa it is not vert descriptive as it is.